consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/statemachine/QbftRound.java

```diff
@@ -87,8 +88,13 @@ public QbftRound(
     this.messageFactory = messageFactory;
     this.transmitter = transmitter;
     this.bftExtraDataCodec = bftExtraDataCodec;
+    this.roundTimer = roundTimer;
 
-    roundTimer.startTimer(getRoundIdentifier());
+    if (getRoundIdentifier().getRoundNumber() == 0) {
```

Comment:
should there be an illegalstateexception (or invalid protocol exception) if rounder number is negative? or is it already sanitized above in the chain?

Reply:
The round number always starts from 0 and is only ever incremented by 1. Don't think we need check that the number isn't negative here. The QbftBlockHeightManager should ensure that we create round change with the correct number.

Reply:
👍🏼